JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsScreen.kt

```diff
@@ -287,22 +282,19 @@ private fun TabWithTopics(
 /**
  * Display a sectioned list of topics
  *
- * @param windowSize (state) the current window size class
  * @param sections (state) topics to display, grouped by sections
  * @param selectedTopics (state) currently selected topics
  * @param onTopicSelect (event) request a topic+section selection be changed
  */
 @Composable
 private fun TabWithSections(
-    windowSize: WindowSize,
     sections: List<InterestSection>,
     selectedTopics: Set<TopicSelection>,
     onTopicSelect: (TopicSelection) -> Unit
 ) {
-    val columns = remember(windowSize) { if (windowSize == WindowSize.Compact) 1 else 2 }
-
     // TODO: Optimize by manually positioning items, rather than using BoxWithConstraints
     BoxWithConstraints {
+        val columns = if (maxWidth < 600.dp) 1 else 2
```

Comment:
Just want to highlight this change: Since we're not quite following the designs with this consolidation, I think it makes sense to make this a local decision again. If/when we re-add the medium layout, we can probably return to the original logic.

Reply:
I'm working on removing BWC from the interests screen, you'll have a PR coming your way soon :D

Reply:
Created #665 as a draft at the moment, but feel free to take a look